From 9453032d6883b48a46ec4ef3f0fe21140d9bd2b5 Mon Sep 17 00:00:00 2001
From: Evgeny Budilovsky <budevg@gmail.com>
Date: Thu, 30 Jan 2020 19:43:34 +0200
Subject: [PATCH] local patch

---
 src/frame.c | 6 +++---
 src/xfns.c  | 4 ++--
 2 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/src/frame.c b/src/frame.c
index 88d6f22fc0..1e2b50253d 100644
--- a/src/frame.c
+++ b/src/frame.c
@@ -4738,7 +4738,7 @@ gui_set_vertical_scroll_bars (struct frame *f, Lisp_Object arg, Lisp_Object oldv
 	 However, if the window hasn't been created yet, we shouldn't
 	 call set_window_size_hook.  */
       if (FRAME_NATIVE_WINDOW (f))
-	adjust_frame_size (f, -1, -1, 3, 0, Qvertical_scroll_bars);
+	adjust_frame_size (f, -1, -1, 3, 0, Qnil);
 
       SET_FRAME_GARBAGED (f);
     }
@@ -6160,7 +6160,7 @@ See the command `menu-bar-mode' for a description of this minor mode.
 Setting this variable directly does not take effect;
 either customize it (see the info node `Easy Customization')
 or call the function `menu-bar-mode'.  */);
-  Vmenu_bar_mode = Qt;
+  Vmenu_bar_mode = Qnil;
 
   DEFVAR_LISP ("tab-bar-mode", Vtab_bar_mode,
                doc: /* Non-nil if Tab-Bar mode is enabled.
@@ -6177,7 +6177,7 @@ Setting this variable directly does not take effect;
 either customize it (see the info node `Easy Customization')
 or call the function `tool-bar-mode'.  */);
 #ifdef HAVE_WINDOW_SYSTEM
-  Vtool_bar_mode = Qt;
+  Vtool_bar_mode = Qnil;
 #else
   Vtool_bar_mode = Qnil;
 #endif
diff --git a/src/xfns.c b/src/xfns.c
index 276ea1c393..d0392ce8df 100644
--- a/src/xfns.c
+++ b/src/xfns.c
@@ -3936,9 +3936,9 @@ This function is an internal primitive--use `make-frame' instead.  */)
                          NULL, NULL, RES_TYPE_NUMBER);
   gui_default_parameter (f, parms, Qvertical_scroll_bars,
 #if defined (USE_GTK) && defined (USE_TOOLKIT_SCROLL_BARS)
-                         Qright,
+                         Qnil,
 #else
-                         Qleft,
+                         Qnil,
 #endif
                          "verticalScrollBars", "ScrollBars",
                          RES_TYPE_SYMBOL);
-- 
2.23.0

