From bdb46e28c5b8f459279783418587f8b9af99c8cb Mon Sep 17 00:00:00 2001
From: Evgeny Budilovsky <budevg@gmail.com>
Date: Tue, 11 Aug 2020 20:36:14 +0300
Subject: [PATCH] local patch

---
 src/frame.c | 6 +++---
 src/xfns.c  | 4 ++--
 2 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/src/frame.c b/src/frame.c
index 4dd8bb1804..29bbd1ab33 100644
--- a/src/frame.c
+++ b/src/frame.c
@@ -4740,7 +4740,7 @@ gui_set_vertical_scroll_bars (struct frame *f, Lisp_Object arg, Lisp_Object oldv
 	 However, if the window hasn't been created yet, we shouldn't
 	 call set_window_size_hook.  */
       if (FRAME_NATIVE_WINDOW (f))
-	adjust_frame_size (f, -1, -1, 3, 0, Qvertical_scroll_bars);
+	adjust_frame_size (f, -1, -1, 3, 0, Qnil);
 
       SET_FRAME_GARBAGED (f);
     }
@@ -6163,7 +6163,7 @@ syms_of_frame (void)
 Setting this variable directly does not take effect;
 either customize it (see the info node `Easy Customization')
 or call the function `menu-bar-mode'.  */);
-  Vmenu_bar_mode = Qt;
+  Vmenu_bar_mode = Qnil;
 
   DEFVAR_LISP ("tab-bar-mode", Vtab_bar_mode,
                doc: /* Non-nil if Tab-Bar mode is enabled.
@@ -6180,7 +6180,7 @@ syms_of_frame (void)
 either customize it (see the info node `Easy Customization')
 or call the function `tool-bar-mode'.  */);
 #ifdef HAVE_WINDOW_SYSTEM
-  Vtool_bar_mode = Qt;
+  Vtool_bar_mode = Qnil;
 #else
   Vtool_bar_mode = Qnil;
 #endif
diff --git a/src/xfns.c b/src/xfns.c
index f9a00a6daf..8b266b2e7f 100644
--- a/src/xfns.c
+++ b/src/xfns.c
@@ -3942,9 +3942,9 @@ DEFUN ("x-create-frame", Fx_create_frame, Sx_create_frame,
                          NULL, NULL, RES_TYPE_NUMBER);
   gui_default_parameter (f, parms, Qvertical_scroll_bars,
 #if defined (USE_GTK) && defined (USE_TOOLKIT_SCROLL_BARS)
-                         Qright,
+                         Qnil,
 #else
-                         Qleft,
+                         Qnil,
 #endif
                          "verticalScrollBars", "ScrollBars",
                          RES_TYPE_SYMBOL);
-- 
2.25.4

